collectors/auth0/Makefile

```diff
@@ -26,7 +27,9 @@ deploy:
 	aws lambda update-function-code --function-name $(AWS_LAMBDA_FUNCTION_NAME) --zip-file fileb://$(AWS_LAMBDA_PACKAGE_NAME)
 
 upload:
-	aws s3 cp ./$(AWS_LAMBDA_PACKAGE_NAME) s3://$(AWS_LAMBDA_S3_BUCKET)
+	aws s3 cp ./$(AWS_LAMBDA_PACKAGE_NAME) s3://$(AWS_LAMBDA_S3_BUCKET)/packages/lambda/
```

Comment:
Don't we need to push these packages to all the region specific buckets? Same with the other collector specific make files?

Reply:
Right. This is mostly done for test purposes when you'd like to upload artifact into your own private s3 bucket for testing.